sdk/communication/communication-rooms/src/generated/src/operations/participants.ts

```diff
@@ -7,20 +7,24 @@
  */
 
 import { tracingClient } from "../tracing.js";
+import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
```

Comment:
Every change under generated/ folder is autogenerated from the rest api spec.
The rest api spec has not changed, but the autogen versions have changed so it has improved the autogenerated code.